src/main/java/com/google/api/codegen/go/GoGapicContext.java

```diff
@@ -428,6 +435,7 @@ private GoImport createMessageImport(MessageType messageType) {
   public Iterable<String> getTestImports(Interface service) {
     TreeSet<GoImport> thirdParty = new TreeSet<>();
 
+    thirdParty.add(GoImport.create(getClientPackagePath(service)));
     thirdParty.add(GoImport.create("golang.org/x/net/context"));
```

Comment:
Not for this PR, but something to keep in mind. Go 1.7 is adding context to stdlib. This might need to change in the future.